src/Eto/Forms/ThemedControls/ThemedDocumentControlHandler.cs

```diff
@@ -677,7 +677,7 @@ void CalculateTab(ThemedDocumentPageHandler tab, int i, ref float posx)
 			size.Width += textoffset;
 		}
 
-		var closesize = tabDrawable.Height / 2;
+		var closesize = UseFixedTabHeight ? minImageSquareSide : tabDrawable.Height / 2;
```

Comment:
I am not sure if the whole comment is necessary. 
Because as a user, I might want to let the tab hight grow  based on the contained  image but still have the close button rendered as 16 x 16. But now this is not possible. What do you think @ievgen-baida ?

Reply:
That is the whole point, because currently the `X` was tiny in relation to 16x16 close button.

`UseFixedTabHeight` is a new option introduced in 2.8.2 so it should be all right to fix it in 2.8.3.